src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -67,12 +68,18 @@ class ignition::gui::plugins::IgnRenderer::Implementation
   /// \brief Flag to indicate if drop event is dirty
   public: bool dropDirty{false};
 
-  /// \brief Mouse event
+  /// \brief Current mouse event
   public: common::MouseEvent mouseEvent;
 
+  /// \brief A list of mouse events
+  public: std::list<common::MouseEvent> mouseEvents;
+
   /// \brief Key event
   public: common::KeyEvent keyEvent;
 
+  /// \brief Max number of mouse events to process
+  public: const unsigned int kMaxMouseEventSize = 5u;
```

Comment:
Curious about where this number came from

Reply:
I picked a small number but it's still a little arbitrary. Main reason to keep a small queue size is that we don't want to keep too many mouse events in the queue otherwise we'll flood other gui plugins with mouse events that are potentially outdated. I expanded the doc in [f68805b](https://github.com/gazebosim/gz-gui/pull/502/commits/f68805bb7a207eb2b5b51806309db0c4eb43843a)